src/databricks/labs/lsql/dashboard.py

```diff
@@ -1,27 +1,31 @@
 import json
```

Comment:
why renaming this module from plural to singular?..

Reply:
yeah, was not necessary, my brain was starting up 🥴